api-spec/openapi/STAC.yaml

```diff
@@ -328,6 +319,21 @@ components:
           example: 'http://www.geoserver.example/stac/naip/child/catalog.json'
         rel:
           example: child
+        method:
+          type: string
+          enum:
+            - GET
+            - POST
+          default: get
+          description: Specifies the HTTP method that the link expects
+        headers:
+          type: object
+          description: Object key values pairs they map to headers
+          example:
+            Accept:  application/json
```

Comment:
should this be an object structure where the attribute name is the header name, or one like:
"headers" : [ { "name": "Accept", "value": "application/json", 
"name": "Authorization", "value": "Bearer mySpecialBearerToken"
]

Reply:
Can there be multiple headers with the same name?

Reply:
I was looking at other tools and it seems the common way of specifying them is with an object rather than an array:
https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch#Headers
https://github.com/request/request#custom-http-headers

I thought it would be convenient to just pass it is rather than restructure it.

Reply:
Agreed an array is more correct for headers.  Many libs accept an object for convenience.  But I don't think it is too onerous if we require that people provide an array.